src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,61 @@
+import styled from '@emotion/styled'
+import { FiChevronLeft, FiUser } from 'react-icons/fi'
+import { useNavigate } from 'react-router-dom'
+
+export function Navbar() {
+  const navigate = useNavigate()
+
+  const handleGoBack = () => {
+    navigate(-1)
+  }
+
+  const handleLoginClick = () => {
+    navigate('/login')
+  }
```

Comment:
```suggestion
  const goBack = () => {
    navigate(-1)
  }

  const goLogin = () => {
    navigate('/login')
  }
```

이 함수가 하는 일로 함수의 이름을 지어주면 좋답니다 ㅎㅎ
지금은 이벤트를 다루고 있지 않아서 handle을 다 생략해주면 어떨까요?